.github/workflows/pull-request.yml

```diff
@@ -128,27 +135,34 @@ jobs:
     name: Build Website
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - name: Setup Python
-        uses: actions/setup-python@v2
-        with:
-          python-version: '3.8'
-      - name: Upgrade pip
-        run: |
-          # install pip=>20.1 to use "pip cache dir"
-          python3 -m pip install --upgrade pip
-      - name: Get pip cache dir
-        id: pip-cache
-        run: echo "::set-output name=dir::$(pip cache dir)"
-      - name: Cache dependencies
-        uses: actions/cache@v2
-        with:
-          path: ${{ steps.pip-cache.outputs.dir }}
-          key: ${{ runner.os }}-pip-${{ hashFiles('site/requirements.txt') }}
-          restore-keys: |
-            ${{ runner.os }}-pip-
-      - name: Install dependencies
-        run: python3 -m pip install -r ./site/requirements.txt
-      - name: Generate Static Site
-        run: mkdocs build
-        working-directory: ./site
+
+    ### BEGIN runner setup
+    - name: Update /etc/hosts
```

Comment:
apologies for being repetitive: I don't personally think we need this line at the start of every run. Or am I forgetting why we added it?